src/payment/strategies/ppsdk/ppsdk-payment-processor.ts

```diff
@@ -3,7 +3,7 @@ import { PPSDKPaymentMethod } from '../../ppsdk-payment-method';
 
 export interface ProcessorSettings {
     paymentMethod: PPSDKPaymentMethod;
-    payment: OrderPaymentRequestBody | undefined;
+    payment?: OrderPaymentRequestBody;
```

Comment:
Little type fix to stop `undefined` from having to be passed in explicitly as a param